Source/WebKit/WebProcess/WebPage/WebPage.messages.in

```diff
@@ -822,4 +822,6 @@ GenerateSyntheticEditingCommand(enum:uint8_t WebKit::SyntheticEditingCommandType
     FrameNameWasChangedInAnotherProcess(WebCore::FrameIdentifier frameID, String frameName)
 
     IsEditingCommandEnabled(String commandName) -> (bool result) Synchronous
+
+    SetPermissionLevelForTesting(String origin, bool allowed)
```

Comment:
This message will be received after TestRunner::grantWebNotificationPermission returns, so it won't be immediately granted like it was before.  If that's a problem, we could make testRunner.grantWebNotificationPermission return a promise and wait for all processes to have the permission granted, but tests seem to pass like this so it may be fine.